tests/unit/amo/reducers/testCarousels.js

```diff
@@ -0,0 +1,86 @@
+import knuthShuffle from 'knuth-shuffle';
+
+import carouselsReducer, {
+  initialState,
+  setCarouselOrder,
+} from 'core/reducers/carousels';
+
+
+describe(__filename, () => {
```

Comment:
Nice, I like this idea, but would it break if we were running tests in a browser in the future? Just I think it's likely that it may be possible to run tests in a browser at some point in the future.

Reply:
Well, we do this every where already, so maybe, but at that point we'll need to deal with it 😉